packages/vhd-lib/Vhd/VhdAbstract.js

```diff
@@ -334,4 +334,25 @@ exports.VhdAbstract = class VhdAbstract {
     stream.length = footer.currentSize
     return stream
   }
+
+  /*
+   * check if all the data of a child are already contained in this vhd
+   */
+
+  async contains(child) {
```

Comment:
More explicit name, which make it more obvious that this is not a light operation:

```suggestion
  async containsAllBlockOf(child) {
```